video/out/vo_gpu_next.c

```diff
@@ -912,6 +912,29 @@ static void apply_crop(struct pl_frame *frame, struct mp_rect crop,
     }
 }
 
+static void update_tm_viz(struct pl_color_map_params *params,
+                          const struct pl_frame *target,
+                          double pts)
+{
+    if (!params->visualize_lut)
+        return;
+
+    // Use right half of sceen for TM visualization, constrain to 1:1 AR
+    const float out_w = fabsf(pl_rect_w(target->crop));
+    const float out_h = fabsf(pl_rect_h(target->crop));
+    const float size = MPMIN(out_w / 2.0f, out_h);
+    params->visualize_rect = (pl_rect2df) {
+        .x0 = 1.0f - size / out_w,
+        .x1 = 1.0f,
+        .y0 = 0.0f,
+        .y1 = size / out_h,
+    };
```

Comment:
This breaks with `--video-rotate=90` for example.

Reply:
Not much we can do about the `--video-rotate` case because the hacky way in which this code is implemented in libplacebo pretty much prevents us from drawing it "properly".

Probably better to fix it inside libplacebo by making the visualization rendering code its own separate output pass that draws on top of the final, rendered image. (This would also fix the part where, as written, we currently draw the visualization _before_ frame caching / interpolation)

Reply:
The best compromise I can come up with is to simply disable the visualizer if `--video-rotate` is non-0. 🤷🏻

Reply:
Fair, just wanted to point it out.